xdsl/backend/csl/print_csl.py

```diff
@@ -549,6 +551,9 @@ def print_block(self, body: Block):
                     ty = cast(csl.PtrType, res.type)
                     use = self._var_use(res, ty.constness.data.value)
                     self.print(f"{use} = &{name.string_value()};")
+                case csl.DirectionOp(dir=d, res=res):
+                    use = self._var_use(res, "const")
```

Comment:
nit: `"const"` is the default introducer for `_var_use`, so you could drop it if you want to
```suggestion
                    use = self._var_use(res)
```